community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/Configuration.java

```diff
@@ -43,7 +43,7 @@
      * database directory of the imported database, i.e. <into>/bad.log.
      */
     String BAD_FILE_NAME = "bad.log";
-    long MAX_PAGE_CACHE_MEMORY = mebiBytes( 240 );
+    long MAX_PAGE_CACHE_MEMORY = mebiBytes( 480 );
```

Comment:
What we generally see is that the smaller the page cache is the faster the importer goes, down to a certain point of course. The smaller it is the higher the chance the background flushing will hit a nice wave of sequential I/O.